src/PerfView/GcStats.cs

```diff
@@ -1051,7 +1104,21 @@ private static string PrintCondemnedReasonsToHtml(TraceGC gc)
             new string[] {"Compacting<BR/>Full<BR/>GC", "Last GC we trigger before we throw OOM"},
             new string[] {"Small<BR/>Heap", "Heap is too small for doing a background GC and we do a blocking one instead"},
             new string[] {"Ephemeral<BR/>Before<BR/>BGC", "Ephemeral GC before a background GC starts"},
-            new string[] {"Internal<BR/>Tuning", "Internal tuning"}
+            new string[] {"Internal<BR/>Tuning", "Internal tuning"},
+            new string[] {"Max Generation Budget", "We are in high memory load situation and have consumed enough max generation budget, so we decided to do a full GC"},
+            new string[] {"Avoid unproductive<BR>Full GC", "This happens when the GC detects previous attempts to do a full compacting GC is not making progress and therefore reduce to Gen1"},
```

Comment:
if you look above you'll notice that all words in the title start with a capital letter, can we please follow that convention? also I would use "HardLimit" instead of "hard limit" in the titles (not in the description) just because this is what you set with the config.

otherwise LGTM.